functions.py

```diff
@@ -285,6 +289,8 @@ def get_jira_set(blockers):
 			print("Error getting jira IDs from blockers file for job {}: {}".format(job, e))
 			continue
 
+	# discard ticket_id value of 0 from set if present as this is not a valid ticket
+	jira_set.discard(0)
```

Comment:
You may want to move this logic into the actual loop when you're building up the set of Jira ticket IDs. That way, you avoid processing the set twice, but realistically, the overhead for making this call to discard a single element from a set shouldn't be that expensive.